homeassistant/components/plugwise/climate.py

```diff
@@ -130,13 +130,13 @@ def hvac_action(self) -> HVACAction | None:
         if control_state == "off":
             return HVACAction.IDLE
 
-        hc_data = self.coordinator.data.devices[
-            self.coordinator.data.gateway["heater_id"]
-        ]
-        if hc_data["binary_sensors"]["heating_state"]:
-            return HVACAction.HEATING
-        if hc_data["binary_sensors"].get("cooling_state"):
-            return HVACAction.COOLING
+        heater: str | None = self.coordinator.data.gateway["heater_id"]
+        if heater is not None:
```

Comment:
It is a str or None, so it is safe to just do:

```suggestion
        if heater:
```